modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -98,5 +90,38 @@ def upstream_service_details(service_name, health_checker, time)
         }
       }
     end
+
+    def decision_reviews_versions
+      if Settings.modules_appeals_api.documentation.path_enabled_flag
+        [
+          decision_reviews_v1.merge({ status: VERSION_STATUS[:previous] }),
+          decision_reviews_v2
+        ]
+      else
+        [
+          decision_reviews_v1
+        ]
+      end
+    end
+
+    def decision_reviews_v1
+      {
+        version: '1.0.0',
+        internal_only: true,
+        status: VERSION_STATUS[:current],
+        path: '/services/appeals/docs/v1/decision_reviews',
+        healthcheck: '/services/appeals/v1/healthcheck'
+      }
+    end
+
+    def decision_reviews_v2
+      {
+        version: '2.0.0',
+        internal_only: true,
+        status: VERSION_STATUS[:current],
+        path: '/services/appeals/docs/v2/decision_reviews',
+        healthcheck: '/services/appeals/v1/healthcheck'
```

Comment:
probably want to have a v2 healthcheck url here, even if it 100% maps to the same v1 logic

Reply:
That's a good point!